herzog/parser.py

```diff
@@ -92,12 +94,14 @@ def _validate_cell(cell_lines: List[str], line_number: Optional[int]=None):
         except SyntaxError:
             raise SyntaxError(f"line {line_number_str}")
 
+def parse_cell_type(s: str) -> str:
+    return s.strip()[len("with herzog.Cell("):-len('):')].strip().strip('"').strip("'").strip()
+
 def parse_cells(raw_lines: TextIO) -> Generator[HerzogCell, None, None]:
     rlines = _RewindableIterator(raw_lines)
     for line in rlines:
         if "with herzog.Cell" in line:
-            cell_type_string = line.strip()[len("with herzog.Cell"):].strip('"():')
-            cell_type = CellType[cell_type_string]
+            cell_type = CellType[parse_cell_type(line)]
```

Comment:
What’s the purpose of the convenience method? Was there a bug, or is this a purely cosmetic change?

In either case this should be split out into a separate PR.

Reply:
There was a bug.  Herzog breaks if you use single quotes.

Reply:
Now fixed in #22.